autoload/db/adapter/sqlserver.vim

```diff
@@ -42,11 +42,11 @@ endfunction
 
 function! db#adapter#sqlserver#interactive(url) abort
   let url = db#url#parse(a:url)
-  return ['sqlcmd', '-S', s:server(url)] +
+  return ['env', 'SQLCMDPASSWORD=' . get(url, 'password'), 'sqlcmd', '-S', s:server(url)] +
```

Comment:
Use `get(url, 'password', '')`. Otherwise the default password will be `0`.

Reply:
sure :+1: